### ISSUE TITLE
[Discussion] Group multiple use, import, alias, require calls

### ISSUE THREAD

**Issue by rgreenjr** [lpil/exfmt#64]
**Feature Proposal**

Per the [style guide's section on modules](https://github.com/lexmag/elixir-style-guide#module-layout), multiple ```use```, ```import```, ```alias```, ```require``` calls should be grouped together and separated with newlines.

Example:

```elixir
defmodule App do
  use GenServer
  import Bitwise
  import Kernel, except: [length: 1]
  alias Mix.Utils
  alias MapSet, as: Set
  require Logger
end
```

should be formatted as:

```elixir
defmodule App do
  use GenServer

  import Bitwise
  import Kernel, except: [length: 1]

  alias Mix.Utils
  alias MapSet, as: Set

  require Logger
end
```

I'm happy to work on this if accepted.

**Comment by rgreenjr** [comment]
@lpil What are your thoughts on this?

**Comment by lpil** [comment]
Hey Ron. Sorry for the slow reply :)

Looks like a good addition to me! We could modify `Ast.insert_empty_lines/1` to put lines between each group.

https://github.com/lpil/exfmt/blob/master/lib/exfmt/ast.ex#L49

**Comment by rgreenjr** [comment]
Great, I'll work on it this weekend.

**Comment by rgreenjr** [comment]
Implemented in #77
